services/user/Supervisor/ui/src/supervisor.ts

```diff
@@ -82,6 +88,8 @@ export class Supervisor implements AppInterface {
     }
 
     private async preload(plugins: QualifiedPluginId[]) {
+        await getChainId();
```

Comment:
Did you intend to send a second request if the first hasn't finished yet?

Reply:
Nope, fixed